### DISCUSSION TITLE
TextStyle does not apply for non-latin characters

### DISCUSSION THREAD

**Discussion by rstm-sf** [QuestPDF/QuestPDF#483] [Q&A]
![image](https://user-images.githubusercontent.com/2669927/217469753-820cce46-204b-4fbe-99ea-b3e22561d08c.png)

Why might this be happening?

**Comment by rstm-sf** [comment]
It turns out that the default style is somehow superimposed on top? Judging by the strikethrough text

**Comment by rstm-sf** [comment]
Solved the problem by setting the default font yet in addition to the fallback:

`TextStyle.Default.Fallback(y => y.FontFamily("Inter"))`
to
`TextStyle.Default.FontFamily("Inter").Fallback(y => y.FontFamily("Inter"))`

**Comment by MarcinZiabek** [comment]
I understand that in your example:
1) Entire first line is bold
2) Entire second line is italic
3) Entire third line is strike-trough

Is that correct? 

Also, in the initial example, you are using `Inter` as a fallback font. What is the default font? (library default Lato)?

**Reply by rstm-sf** [reply]
Hello, yes it is

**Reply by rstm-sf** [reply]
Can see here changes https://github.com/Relorer/HTMLToQPDF/pull/3/files#diff-fa67b9898be30f849629f8a23f6e337766fe360b08336bdc9981b8e9d64a44b0R21

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

It is also worth noting that the text rendering engine has since undergone a significant overhaul, which likely addresses the behavior you described.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
